docs/reference/esql/functions/kibana/definition/match_operator.json

```diff
@@ -1,6 +1,7 @@
 {
   "comment" : "This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.",
   "type" : "operator",
+  "operator" : "MATCH",
```

Comment:
Should this be `"operator": ":"`?

Reply:
For clarity, we should update the examples from this file to reflect that `:` as well 
`FROM books \n| WHERE MATCH(author, \"Faulkner\")\n| KEEP book_no, author \n| SORT book_no \n| LIMIT 5;
`
to 
`FROM books \n| WHERE author:Faulkner`

Reply:
Yeah, it should be `:`. The whole point is to give y'all the glyphs that trigger the operators and `:` is the one for MATCH. Fix incoming.